modules/rosa-edit-objects.html

```diff
@@ -0,0 +1,761 @@
+<!DOCTYPE html>
```

Comment:
Not sure if you wanted us to review this or not, but a quick scan and the html looks good.

Reply:
Thank you! This is an odd one, but I think it's okay.